main.rb

```diff
@@ -2,28 +2,4 @@
 
```

Comment:
- [ ] Great job so far. please, what the previous reviewer meant was that, in your `main.rb` file you should add a method `main` like mentioned in your project requirement. this method will contain a list of options for the user to chose and how to quit like you did and finally at the bottom of the file the main method is invoked. 
you are almost done. keep up the good work.
eg 
```ruby
def main
  app = App.new()
...
...
...

end
end


main()
```

Reply:
Cant believe i forgot this, thanks for the review